CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
 * New eslint-plugin-jest with no-disabled-tests, no-focuses-tests and no-identical-title rules and default configuration and globals.
 * New expect.stringContaining asymmetric matcher.
 * New feature to make manual mocks with nested folders work. For example `__mocks__/react-native/Library/Text.js` will now work as expected.
+* New feature to re-run tests through the notification when using `--notify`.
```

Comment:
This line doesn't make sense if you don't know that "the notification" is (like I don't)